components/ws-manager-bridge/src/bridge.ts

```diff
@@ -518,12 +518,12 @@ export class WorkspaceManagerBridge implements Disposable {
                             instance.stoppingTime &&
                             durationLongerThanSeconds(Date.parse(instance.stoppingTime), 10))
                     ) {
-                        log.info("Logging to validate #12902. Should mark as stopped in database.", {
-                            instanceId,
-                            workspaceId: instance.workspaceId,
-                            installation,
-                            phase,
-                        });
+                        log.info(
```

Comment:
Interesting that our typechecking is not catching these..

Reply:
Both are correct, type-wise and semantically. :shrug: